Replaced one paper and updated page numbers as appropriate

paulratdollabs opened the pull request:
Replaced Thorisson paper with the latest version. Updated page numbers in followon papers.  Updated the bibteX file to reflect the changed page numbers.

paulratdollabs committed:
Replaced one paper and updated page numbers as appropriate

Replaced Thorisson paper with the latest version. Updated page numbers in followon papers.  Updated the bibteX file to reflect the changed page numbers.

lawrennd commented on IWSSL2020.bib:
```diff
@@ -143,7 +143,7 @@ @InProceedings{lieberman20
                   Systems},
   author =	 {Lieberman, Henry},
-  pages =	 {62--71},
+  pages =	 {71--79},
```
Did this paper get shorter, shouldn't this be 80?

paulratdollabs replied:
Sigh, yes, you are right.  I'll need to make the fixes and try again.

lawrennd reviewed (commented):
Hi Paul, did one of the papers also get shorter? I notice a page got dropped for Lieberman in the total (but didn't double check the PDF).

paulratdollabs committed:
Fixed off by one page number error.

Fixed off by one counting error and the propagation effects on page numbering.  Double checked this time!

paulratdollabs commented:
Thank you for pointing out that error in page numbering.  I have fixed it now and double checked it this time.

paulratdollabs commented:
Let's try this pull request again.  page number errors have been fixed.

lawrennd reviewed (approved):
Looks good.
